src/components/accordion-disclosure/index.tsx

```diff
@@ -11,18 +11,39 @@ import {
 	useAccordionDisclosureContext,
 	AccordionDisclosureContext,
 } from './accordion-disclosure-context'
+import useHover from 'hooks/use-hover'
 
 const AccordionDisclosure = ({
 	children,
 	description,
 	initialOpen,
 	title,
+	isFirstItem,
+	isLastItem,
+	activatorHeadingLevel,
 }: AccordionDisclosureProps) => {
+	const [hoverRef, isHovered] = useHover<HTMLButtonElement>()
```

Comment:
Do we have to use the hook because it's hard to capture `:hover` on the right component?

Reply:
Yep, exactly!

When `button` is hovered, I think we want the box-shadow on the enclosing section to change. So `:hover` won't work in CSS since we could only target elements nested within `button`.

For context, the previous implementation had a box-shadow on both `button` and the enclosing section. This kind of worked, but had two issues IMO:
- no hover style when open
- box-shadow from `button` creates an intersecting line with content, which looks a little odd during the animation / hover change as the content opens but the box-shadow on button is still present


https://user-images.githubusercontent.com/4624598/206761864-d0db93b4-0015-48b0-82be-03db61ac0150.mp4

Reply:
okay that makes sense! I like that we have over styles when opened now too. Maybe we could leave a comment here for why we're doing hover like this as a breadcrumb.

Reply:
great call on the breadcrumb comment 💯  ✅ added that in!